packages/markdown-pdf/src/PdfTransformer.js

```diff
@@ -17,6 +17,8 @@
 const parsePdf = require('easy-pdf-parser').parsePdf;
 const extractPlainText = require('easy-pdf-parser').extractPlainText;
 const CiceroMarkTransformer = require('@accordproject/markdown-cicero').CiceroMarkTransformer;
+const markdownpdf = require('markdown-pdf');
```

Comment:
This module includes PhantomJS as a dependency. PhantomJS is no-longer supported (https://github.com/ariya/phantomjs/issues/15344). I suggest using a pupetter-based rendered

Reply:
Yes, phantomjs and puppeteer both are headless chromium. As phantomjs development is suspended so I will move to puppeteer. Thanks you for your suggestion.

Reply:
I would rather that we *not* use a PDF renderer that relies on Chrome. Can we use something like `pdfkit` (or similar) that is pure Node.js?